### ISSUE TITLE
Transfer Funds

### ISSUE THREAD

**Issue by RickKraut** [phpBB-Studio/AdvancedPointsSystem#8]
I figure it's totally reasonable for you to redirect this back toward's 'ASS', as it's just easier for you guys to leave it at that, but the ability to transfer funds within this extension might be a nice, especially since you have to 
individually transfer items one at a time while also having to deal with them not stacking.

**Comment by 3D-I** [comment]
I think this will be adressed to the `Bank` extension.

No ETA.

**Comment by mrgoldy** [comment]
No, bank is something else (where users store their points and receive interest).
This is about 'donations'. Which is handled by the Shop's item type "Points". 
Where you can gift that item to other users.

If you configure the item properly, there is no need for stacking at all.
Item can be used once and deleted immediatly afterwards. 

Anyways, the point of releases it to test the current things and not feature requests within the first day of release.
